pkg/kubelet/images/image_manager.go

```diff
@@ -154,11 +154,7 @@ func applyDefaultImageTag(image string) (string, error) {
 	_, isTagged := named.(dockerref.Tagged)
 	_, isDigested := named.(dockerref.Digested)
 	if !isTagged && !isDigested {
-		named, err := dockerref.WithTag(named, parsers.DefaultImageTag)
-		if err != nil {
-			return "", fmt.Errorf("failed to apply default image tag %q: %v", image, err)
-		}
-		image = named.String()
+		image = image + ":" + parsers.DefaultImageTag
```

Comment:
if you have time before this merges, please leave a note explaining why you are not using named in this context so its not updated in a future pr inadvertently.